Dockerfile.rhel

```diff
@@ -32,9 +32,8 @@ RUN make build
 #--------------------------------------------------------------------
 
 # On CI , use
```

Comment:
I would remove this line as well.

Reply:
done https://github.com/redhat-developer/service-binding-operator/pull/316/commits/4c4b2a8ddc0919ec5b401c8b3593e8d91417a15c